packages/peregrine/src/RestApi/Magento2/M2ApiRequest.js

```diff
@@ -191,15 +191,25 @@ class M2ApiRequest {
                             // includes the original context of the request,
                             // and formats the server response.
                             .then(bodyText => {
-                                throw new M2ApiResponseError({
```

Comment:
@zetlen `M2ApiResponseError` builds up a nice pretty message out but it looks more formatted for the terminal than for the front end. The front end really just needs the error `message`.

Can we work together to find a solution here that works for both terminal and UI?

Reply:
I ended up exposing the base error `message` as `baseMessage`. This keeps the current functionality and adds a simple property to access the original `message` easily if needed.